docs/android/integration/integration-steps.md

```diff
@@ -7,12 +7,12 @@ description: Integration steps to add Embrace to your Android application
 # Integration Steps
 
 1. [**Login to the Embrace dashboard.**](/android/integration/login-embrace-dashboard) Your data is available in the dashboard. It is important that you have this open while working on the integration to monitor progress.
-1. [**Adding the Embrace SDK.**](/android/integration/add-embrace-sdk) We'll be adding the Embrace SDK to your project
+2. [**Adding the Embrace SDK.**](/android/integration/add-embrace-sdk) We'll be adding the Embrace SDK to your project
```

Comment:
@vitaliyf would the linter catch number-ordering issues in the future? always preferred 1.1.1.1. as opposed to sequential since it doesn't make you look bad if you mess up a number